recipes/swig/all/test_package/CMakeLists.txt

```diff
@@ -1,16 +1,24 @@
 cmake_minimum_required(VERSION 3.1)
 project(PackageTest C)
+enable_testing()
 
 include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
-conan_basic_setup()
+conan_basic_setup(NO_OUTPUT_DIRS)
+
+find_program(PYTHON_EXECUTABLE NAMES python3 python REQUIRED)
```

Comment:
Can you use `FindPythonInterp` here?
https://cmake.org/cmake/help/latest/module/FindPythonInterp.html

Reply:
https://cmake.org/cmake/help/latest/module/FindPythonInterp.html
> Deprecated since version 3.12

Not sure it's a better option.